### ISSUE TITLE
Seems like "Is this major breaking change?" does nothing (almost)

### ISSUE THREAD

**Issue by charlike** [bahmutov/simple-commit-message#16]
More specifically said, it does, but that adding of `BREAKING CHANGE` to body and title seems almost useless. Or at least not make sense to output that prompt every time for every kind of type.

Not to mention that the wizard (the prompts) doesn't have `break` / `major` and etc. It would be good to at least have `major` because it seems that semantic-release don't care about the wordings, but care only about `major` type, which is awful, but anyway.

**Comment by bahmutov** [comment]
Yeah, I pretty much never use the wizard feature, since I remember the simple convention. So for me it is always `major(foo): changed how foo works` or something like that
